requirements_test.txt

```diff
@@ -4,3 +4,4 @@ optax>=0.0.9
 pytest-xdist
 scikit-learn>=0.24.1
 cvxopt
+pytest-xdist
```

Comment:
See line 4 :)

Reply:
LOL I'm dense

Reply:
removed this line. Let's see if tests pass